packages/stryker/test/unit/TempFolderSpec.ts

```diff
@@ -0,0 +1,84 @@
+import { TempFolder } from '../../src/utils/TempFolder';
+
+import * as sinon from 'sinon';
+import { expect } from 'chai';
+import * as mkdirp from 'mkdirp';
+import * as fs from 'mz/fs';
+
+describe.only('TempFolder', () => {
+  let sandbox: sinon.SinonSandbox;
+
+  beforeEach(() => {
+    sandbox = sinon.sandbox.create();
+  });
+  afterEach(() => sandbox.restore());
+
+  it('TempFolder is presented', () => {
```

Comment:
What exactly are you testing here? This test can never fail, right? If one of the methods is missing, we would get a compile error in our production code.  The benefit of using typescript, is that we don't need tests like these. You can remove it :+1:

In this test, your basically verifying that TypeScript is transpiled correctly.

Reply:
This test will fail if somebody accidently remove/rename this class. This test will force him think about it again.

Reply:
No. That's why we use typescript. Having the test won't hurt of course.